src/aap_eda/utils/__init__.py

```diff
@@ -11,7 +11,36 @@
 #  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 #  See the License for the specific language governing permissions and
 #  limitations under the License.
+from importlib.metadata import PackageNotFoundError, version
+from pathlib import Path
+
+import toml
+
+PYPROJECT_PATH = Path(__file__).parent.parent.parent.parent
 
 
 def str_to_bool(value: str) -> bool:
     return value.lower() in ("yes", "true", "1")
+
+
+def get_eda_version():
+    """Return EDA version as defined in the package or pyproject.toml."""
+    try:
+        return version("aap-eda")
+    except PackageNotFoundError:
+        return get_version_from_file()
```

Comment:
@Dostonbek1 Can this just be "N/A". Are you seeing this only during pytest? In all other scenarios
* Container Images have the package aap-eda package installed
* The RPM packages should have the aap-eda package installed or does it have a different name.

Reply:
I mean it could but as we saw in container instance, if aap-eda package is not available for whatever reason then we would resort to getting it directly from pyproject.toml file.